src/ActionParser.h

```diff
@@ -30,22 +32,23 @@ namespace snowcrash {
     /** Action Definition Type */
     enum ActionType {
         NotActionType = 0,
-        DependentActionType,      /// Action isn't fully defined, depends on parents resource URI
-        CompleteActionType,       /// Action is fully defined including its URI
+        DependentActionType, /// Action isn't fully defined, depends on parents resource URI
+        CompleteActionType, /// Action is fully defined including its URI
         UndefinedActionType = -1
     };
 
     /**
      * Action Section processor
      */
-    template<>
+    template <>
     struct SectionProcessor<Action> : public SectionProcessorBase<Action> {
 
         static MarkdownNodeIterator processSignature(const MarkdownNodeIterator& node,
-                                                     const MarkdownNodes& siblings,
-                                                     SectionParserData& pd,
-                                                     SectionLayout& layout,
-                                                     const ParseResultRef<Action>& out) {
+            const MarkdownNodes& siblings,
```

Comment:
I don't think it's possible to do what you are suggesting with clang-format @pksunkara.

Here's from clang-format docs:

<img width="1061" alt="screen shot 2016-05-19 at 21 43 28" src="https://cloud.githubusercontent.com/assets/44164/15417446/d95645e8-1e0a-11e6-92fe-fd594c2cf0dd.png">